src/sage/schemes/riemann_surfaces/riemann_surface.py

```diff
@@ -4012,107 +4011,110 @@ def vectomat(v):
     return [vectomat(v) for v in mtL if all(a.abs() <= c for a in v[g1 * g2 :])]
 
 
-def alternating_smith_form(M):
+def skew_form(M, transformation=False):
     r"""
-    Given an integer 2m x 2g matrix M, perform skew Smith normal form
-    and permute to get MJM^T = (0, delta; -delta, 0).
+    Compute the skew Smith normal form of a square integer matrix as defined
+    in Theorem IV.1 of Integer Matrices by Newman.
+
+    Given an integer matrix `M`, we compute a unimodular matrix `S` such that
+    `S^T M S` is block diagonal with 2x2 blocks `(0, d_i; -d_i, 0)` with `d_i`
+    being positive integers and return the transformed matrix.
 
     INPUT:
 
-    - ``M`` -- integer 2m x 2g matrix
+    - ``M`` -- square integer matrix
+    - ``transformation`` -- if True, return transformation matrix S
```

Comment:
[The documentation should say what the default value of an input is](https://doc.sagemath.org/html/en/developer/coding_basics.html#the-docstring-of-a-function-content). This applies to your later functions as well.